Submodule common f0c2dc9aa..ed78bee43:
  > valgrind suppressions: ignore any error triggered by */ld*.so
  > gtk-doc-plugins.mak: sort xml inspect files
diff --git i/ext/opencv/MotionCells.cpp w/ext/opencv/MotionCells.cpp
index f85989e11..e75191308 100644
--- i/ext/opencv/MotionCells.cpp
+++ w/ext/opencv/MotionCells.cpp
@@ -52,6 +52,7 @@
 #include <errno.h>
 #include "MotionCells.h"
 #include <opencv2/imgproc/imgproc_c.h>
+#include <opencv2/imgproc/imgproc.hpp>
 
 MotionCells::MotionCells ()
 {
diff --git i/ext/opencv/gsthanddetect.cpp w/ext/opencv/gsthanddetect.cpp
index 60fd5be72..aadbc03de 100644
--- i/ext/opencv/gsthanddetect.cpp
+++ w/ext/opencv/gsthanddetect.cpp
@@ -63,6 +63,7 @@
 /* element header */
 #include "gsthanddetect.h"
 #include <opencv2/imgproc/imgproc_c.h>
+#include <opencv2/imgproc.hpp>
 
 GST_DEBUG_CATEGORY_STATIC (gst_handdetect_debug);
 #define GST_CAT_DEFAULT gst_handdetect_debug
diff --git i/ext/opencv/gstsegmentation.cpp w/ext/opencv/gstsegmentation.cpp
index d52476e7d..fd3abb507 100644
--- i/ext/opencv/gstsegmentation.cpp
+++ w/ext/opencv/gstsegmentation.cpp
@@ -92,6 +92,7 @@
 
 #include "gstsegmentation.h"
 #include <opencv2/imgproc/imgproc_c.h>
+#include <opencv2/imgproc.hpp>
 
 GST_DEBUG_CATEGORY_STATIC (gst_segmentation_debug);
 #define GST_CAT_DEFAULT gst_segmentation_debug
diff --git i/ext/opencv/gsttemplatematch.cpp w/ext/opencv/gsttemplatematch.cpp
index f39208dc2..50197f23d 100644
--- i/ext/opencv/gsttemplatematch.cpp
+++ w/ext/opencv/gsttemplatematch.cpp
@@ -64,6 +64,7 @@
 #include "../../gst-libs/gst/gst-i18n-plugin.h"
 #include "gsttemplatematch.h"
 #include <opencv2/imgproc/imgproc_c.h>
+#include <opencv2/imgproc.hpp>
 
 GST_DEBUG_CATEGORY_STATIC (gst_template_match_debug);
 #define GST_CAT_DEFAULT gst_template_match_debug
